lib/acts_as_rdfable/serializers/oai_etdms.rb

```diff
@@ -17,14 +17,13 @@ def self.serialize(object, xml:)
       oai.etd_ms(:language, object.language)  if object.language.present?
       oai.etd_ms(:rights, object.rights) if object.rights.present?
 
-      if object.degree.present?
```

Comment:
So this https://github.com/ualbertalib/jupiter/blob/63f6b1ba1e47c9c021660bf55b1807bfda828853/app/decorators/metadata/oai_etdms/thesis_decorator.rb#L57 replaces the `present?` check?

Reply:
Correct, needed to do this because object.degree (degree name) is a mandatory field so it needs a default